index.js

```diff
@@ -1,17 +1,30 @@
 'use strict';
 var fs = require('fs');
+
 var isDocker;
 
-function check() {
+function hasDockerEnv() {
 	try {
-		fs.statSync('/.dockerinit');
 		fs.statSync('/.dockerenv');
 		return true;
 	} catch (err) {
 		return false;
 	}
 }
 
+function hasDockerCGroup() {
+	try {
+		var contents = fs.readFileSync('/proc/1/cgroup');
```

Comment:
According to http://tuhrig.de/how-to-know-you-are-inside-a-docker-container/, it's better to use `self` instead of `1`.